conans/test/functional/toolchains/microsoft/test_msbuild.py

```diff
@@ -1,13 +1,10 @@
 import os
 import platform
-import shutil
 import textwrap
-import unittest
 
 import pytest
-from parameterized import parameterized, parameterized_class
 
-from conan.tools.microsoft.visual import vcvars_command
+from conan.tools.microsoft.visual import msvc_version_to_vs_ide_version, vcvars_command
```

Comment:
Please do not modify existing tests. There is a risk of breaking some functionality. The feature that you are proposing is a opt-in, so the previous tests shouldn't be modified, and adding a new test that uses the new functionality is the way to go.

Reply:
It's deeply modified because this test was not easily extendable. It seems weird to add a separate test with a risk to have discrepancies between both flavors.
But yeah if you want I can move this to another file, it's just a lot of duplicated code (specially this big vcxproj file).